docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1620,6 +1620,41 @@ its value will be replaced with the provided one.
 }
 --------------------------------------------------
 
+[[size-processor]]
+=== Size Processor
+Sets the size of the current document as it will be rendered as a JSON document.
+
+[[size-options]]
+.Siz Options
+[options="header"]
+|======
+| Name                  | Required | Default      | Description
+| `target`              | no       | `_meta.size` | The target field the size will be stored in
```

Comment:
Well. I choose this one because of this: https://www.elastic.co/guide/en/elasticsearch/reference/5.0/mapping-meta-field.html and I found it could be nice to use it as we are mentioning that in our docs. TBH I first try `_size` but it was rejected as `reserved` name. As we need to leave with both mapper and ingest, I decided not to change the check.